src/components/GithubRedirect/GithubRedirect.tsx

```diff
@@ -1,22 +1,26 @@
 import { Navigate, useLocation, useParams } from 'react-router-dom';
 import { Bullseye, Flex, HelperText, HelperTextItem, Spinner } from '@patternfly/react-core';
+import { useErrorState } from '~/shared/hooks/useErrorState';
 import { PipelineRunLabel } from '../../consts/pipelinerun';
 import { usePipelineRun } from '../../hooks/usePipelineRuns';
-import { HttpError } from '../../k8s/error';
 import { GithubRedirectRouteParams } from '../../routes/utils';
-import ErrorEmptyState from '../../shared/components/empty-state/ErrorEmptyState';
 
 const GithubRedirect: React.FC = () => {
   const { pathname } = useLocation();
   const { ns, pipelineRunName, taskName } = useParams<GithubRedirectRouteParams>();
   const isLogsTabSelected = pathname.includes('/logs');
   const [pr, loaded, error] = usePipelineRun(ns, pipelineRunName);
+  const errorState = useErrorState(error, loaded, 'logs');
```

Comment:
nit: it should be ''pipeline run" instead of "logs", right? given the context where the error is coming from :)